control-plane/subcommand/install-cni/command.go

```diff
@@ -0,0 +1,373 @@
+package installcni
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"sync"
+
+	"github.com/containernetworking/cni/libcni"
+	"github.com/hashicorp/consul-k8s/control-plane/cni/config"
+	"github.com/hashicorp/consul-k8s/control-plane/subcommand/common"
+	"github.com/hashicorp/consul-k8s/control-plane/subcommand/flags"
+	"github.com/hashicorp/go-hclog"
+	"github.com/mitchellh/cli"
+	"github.com/mitchellh/mapstructure"
+)
+
+const (
+	defaultName            = "consul-cni"
```

Comment:
It is the name of the plugin and the name that get placed in the config file. I supposed this should probably go into `./control-plane/cni/config.go` since that is where the config struct is.

Reply:
okay then maybe call this defaultPluginName?

Reply:
although is it a default or just the actual name

Reply:
It is the actual name, nobody can change it.

Reply:
so maybe `pluginName`?